### PR TITLE
Modified the Buildkite configuration file to enable automated FOSSA s…

### PR DESCRIPTION
**Pull request by xinding33**
I'm from FOSSA and I'm working with the Uber OSPO to automate OSS license scanning. In addition to the changes in this PR, we may also need to add an API key as an environmental variable in the build environment. I don't work with Buildkite frequently so I apologize if I'm making the incorrect changes to pipeline.yml. I'm essentially trying to install the FOSSA open source dependency discovery component to the build system and run it after the code has been built. Open to suggestions on how best to add the FOSSA tool to your existing CI/CD pipeline.

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -51,3 +51,10 @@ steps:
             - "./:/work"
     agents:
       queue: workers
+    # Wait until all the previous tests have completed
+    - wait
```
Comment (peter-edge) [inline .buildkite/pipeline.yml]: Remove this wait

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -51,3 +51,10 @@ steps:
...
+    # Run FOSSA to scan for OSS license compliance
+  - name: ':fossa-oss-scan:'
```
Comment (peter-edge) [inline .buildkite/pipeline.yml]: name this `'fossa oss scan'`

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -51,3 +51,10 @@ steps:
...
+    commands:
+      - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | sudo bash"
```
Comment (peter-edge) [inline .buildkite/pipeline.yml]: Remove sudo, no need for it

File: .buildkite/pipeline.yml [hunk 4]
```diff
@@ -51,3 +51,10 @@ steps:
             - "./:/work"
     agents:
       queue: workers
+    # Wait until all the previous tests have completed
+    - wait
+    # Run FOSSA to scan for OSS license compliance
```
Comment (peter-edge) [inline .buildkite/pipeline.yml]: Remove comment, the name is self-documenting

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (peter-edge) [inline .buildkite/pipeline.yml]: Please do this inside the `buildkite` docker container, similar to other tests

**Comment by peter-edge** [comment]
@xinding33 I can just do a PR and do these changes if it's easier for you

**Comment by peter-edge** [comment]
@xinding33 I put together #448 to replace this, however neither this nor #448 have `FOSSA_API_KEY` set, so both will fail. How do I set this?

**Comment by xinding33** [comment]
Perfect! Sorry, I'm working with a lot of Uber open source teams at the same time so it's taking a little longer to respond. You'll receive an internal email soon detailing the steps to add the FOSSA_API_KEY.

**Comment by peter-edge** [comment]
I'm going to close this in favor of #448, will work to get that in.

**Pull request closed without merging**